apps/code-examples/src/app/code-examples/lookup/lookup/async/lookup-async-demo.component.html

```diff
@@ -14,8 +14,8 @@
         idProperty="name"
         selectMode="single"
         [ariaLabelledBy]="nameLabel.id"
+        [data]="people"
```

Comment:
I thought `data` wasn't necessary when using `searchAsync`?

Reply:
It isn't. I messed this up in a merge conflict. My bad! Resolved in [fix improper handling of merge conflicts](https://github.com/blackbaud/skyux/pull/381/commits/0d93d4952b71c525f163a9b9fd0f84537db2d2bf)